### ISSUE TITLE
Revamp - Quest Areas

### ISSUE THREAD

**Issue by Valo56** [zunath/SWLOR_NWN#1391]
Basically, a series of instanced quest areas for groups to tackle. For lower level quests, individuals could solo if desired so low level players can experience more content on their own if there are few others to group with at a similar power level. This idea is shamelessly stolen from servers like EfU, Netheril, City of Arabel, etc.

-You can enter with a group once per day, a server reset lets you re-enter. Alternatively give an 8 hour cooldown if we want players to be able to do it more often. They are intended to be redone with groups but not spammed. 
-For most quests, other than beginner quests on CZ and Viscara, you need a group to enter.
-If you're not part of the quest group you don't get to enter, it's a private group affair. The transitions to the quest areas won't work for you unless you have the quest active. Players can add other people to their quest group with the quest giver NPC though if they need more help or there's a latecomer.
-Only one group can be doing the quest at a time (unless we decide to clone the areas for multiple instances which might be easier idk)
-When the quest is started the NPC's and any treasure spawns in. Once. NPC's don't respawn during the quest.
-The goal is to reach some location, recover some item, defend someone/something, or kill some boss, or some other objective depending on the quest.
-Once successful in the objective (and any secondary objectives you choose to go for) players return for a reward of a sizeable amount of RPXP and credits.

**Comment by Valo56** [comment]
Quest 1 - Jaws of Defeat
Difficulty - Endgame

A Sith raiding party ambushed a Republic supply line. By the time backup arrived it was too late, the valuable goods destined for fringe worlds were destroyed or taken. One of the Sith ships, however, a new design of frigate with considerable firepower, has been crippled. So far it has resisted boarding attempts, though has not self destructed, perhaps as such functionality has been disabled in the battle. Republic Command wants the ship boarded and fully disabled so it can be brought back to a shipyard for disassembly and study. The players will board the ship, landing in one of its hangar bays. From there they will have to take over the ship. There will be several objectives across it, ranging from disabling weapons, missiles, its reactor, and of course the boss at the bridge of the ship.

**Comment by Valo56** [comment]
Quest 2 - Fixer Upper
Difficulty - Beginner, solo possible

The troubles at CZ-220 can only go on for so long. Rogue droids and cascading systems failures will inevitably render the station inoperable. Management is tasking some new arrivals to the station to pay for stay by descending down and clearing out some of the dangers in the lower decks. The player(s) will have to clear out swarms of rogue droids, including the as-yet-unseen maintenance droids that feature a bulkier, stronger frame suited for work in the core of the station. The players' task is primarily to clear out the danger and destroy a corrupted central command droid that Management believes is responsible for many of the problems aboard the ship. Players can earn extra rewards by utilizing scrap metal to patch up bulkheads and electronics to repair damaged systems. (Each damaged section can be attempted to be repaired only once, using either Fabrication for bulkheads or Engineering for systems, giving a chunk of XP to either on success and bonus credits and RPXP on quest turn-in.)

**Comment by Valo56** [comment]
Quest 3 - Breaking a Monopoly
Difficulty - Early Game, solo possible

Viscara is a fledgling colony, still struggling to establish itself and the ongoing war renders self sufficiency a valuable trait as supply lines are constantly in flux. However, for many exotic or specialized materials like medicine there are only a few possible suppliers in the galaxy. This is why the recent pirate raids at the system borders is such a big deal. Shipments of valuable medicinal spices have been taken, and the Ruby Ravens pirate gang has been sitting on its supply selling it off on the black market for exorbitant prices. As the poor suffer and die and even the well off suffer massive setbacks from the all-too-common sickness and injuries in the dangerous colony, support for Czerka and their ability to keep society safe and healthy is plummeting.

As such, Czerka is hiring mercenaries to locate and put an end to the pirate gang's hideout, recover the spices and get paid handsomely in the process. Of course, once you have the spices, what might you do? Will you simply return them to Czerka for them to establish their own monopoly? Perhaps you could find your own fence for the goods for more credits, or find a charitable organization to distribute them. (3 different quest turn-in points.)

**Comment by Valo56** [comment]
Quest 4 - Breaking a Siege
Difficulty - Mid Game, space combat

The war visits a Republic colony a few parsecs from Viscara. A small strikeforce of Sith ships has blocked off all traffic to and from the planet, and orbital strikes are becoming more and more aggressive as they continue to demand tribute and a change in loyalty from the colonists. The settlement's shields won't last forever, and they're beginning to lose hope. At the Viscaran spaceport a colonist who had come to the planet to trade is begging mercenaries to assist, and though the Republic promises help is on the way they are stretched too thin to respond immediately.

Players will have to board their ships and head to the system, where they will encounter several Sith Fighters and an enemy frigate. (If I can figure out how to script this) skilled engineers will be able to repair a damaged space station's weapon systems to help in the fight, as well.

(Intended for at least two player ships of Tier 3 rank.)
